Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoAccount.java

```diff
@@ -209,9 +245,17 @@ public void process(ResultSet rs) throws CentralRepoException, SQLException {
      * @throws CentralRepoException If there is an error in getting the accounts.
      */
     public static Collection<CentralRepoAccount> getAccountsWithIdentifier(String accountIdentifier) throws CentralRepoException {
-       
+
+        String normalizedAccountIdentifier;
+
+        try {
+            normalizedAccountIdentifier = normalizeAccountIdentifier(accountIdentifier);
+        } catch (TskCoreException ex) {
+            throw new CentralRepoException("Failed to normalize account identifier.", ex);
+        }
+
         String queryClause = ACCOUNTS_QUERY_CLAUSE
-                + " WHERE LOWER(accounts.account_unique_identifier) = LOWER('" + accountIdentifier + "')";
+                + " WHERE LOWER(accounts.account_unique_identifier) = LOWER('" + normalizedAccountIdentifier + "')";
```

Comment:
Would it make sense for normalizeAccountIdentifier to take care of putting the  identifier into lowercase?  An email address passed into that method would already be in all lower case as CommunicationsUtils does that.

Reply:
Normalization of other account types may do different things.  I don't know if we should assume it will convert to lower case.  We just had a discussion today to allow for different normalizations for Accounts  (as created by CommunicationsUtils)  Vs. CentralRepoAccounts